lib/cloudwatch-integration.js

```diff
@@ -113,6 +122,13 @@ function retrySubmit(aws, payload, times, cb) {
 }
 
 lib.getToken = function(aws, groupName, streamName, retentionInDays, cb) {
```

Comment:
Here I'm going to probably ask a silly question, so forgive me if it is :D, is the AWS endpoint happy if we just store the token for later use (probably it's even a best practice, I'm not actively using this module in production anymore since a couple years)?

Reply:
Haha, all good. The DescribeLogStreams call has quite a low rate limit, and is a global / account limit (5 tps by default). Amazon let's you increase it a bit but still very easy to hit when you have quite a few servers/containers logging. Their API is more than happy with you using this and the rate limit on Describe suggests it's best practice.

Reply:
Cool!